### PR TITLE
upgrade es to v7.5.1 and use basic auth

### PR DESCRIPTION
**Pull request by marshz9**

### CODE REVIEW

**Comment by marshz9** [comment]
@windmgc plz review

File: sa_tools_core/libs/es.py [hunk 1]
```diff
@@ -9,10 +9,9 @@
 
 
 class ESQuery(object):
-    def __init__(self, es_hosts, index_prefix, doc_type, timestamp_field, index_time_format='%Y.%m.%d'):
-        self.client = Elasticsearch(hosts=es_hosts)
+    def __init__(self, es_hosts, es_user, es_passwd, index_prefix, timestamp_field, index_time_format='%Y.%m.%d'):
```
Comment (windmgc) [inline sa_tools_core/libs/es.py:12]: I think `es_user` and `es_passwd` should have default values

File: sa_tools_core/libs/es.py [hunk 2]
```diff
@@ -9,10 +9,9 @@
...
+        self.client = Elasticsearch(hosts=es_hosts, http_auth=(es_user, es_passwd))
```
Comment (windmgc) [inline sa_tools_core/libs/es.py:13]: Whether `http_auth` should be used can be determined by the values of two vars

**Comment by marshz9** [comment]
@windmgc fixed, PTAL

**Pull request merged**